gcloud/exceptions.py

```diff
@@ -187,6 +187,9 @@ def make_exception(response, content, use_json=True):
     message = payload.get('error', {}).get('message', '')
     errors = payload.get('error', {}).get('errors', ())
 
+    if request_string:
+        message += ' (%s)' % request_string
```

Comment:
I'd prefer `if request_string is not None:` (an explicit test for the parameter default).